tests-mayastor/tests/nexus.rs

```diff
@@ -1,15 +1,16 @@
 #![feature(allow_fail)]
 use testlib::*;
+use types::v0::message_bus::mbus;
 
 #[actix_rt::test]
 async fn create_nexus_malloc() {
     let cluster = ClusterBuilder::builder().build().await.unwrap();
 
     cluster
         .rest_v0()
-        .create_nexus(v0::CreateNexus {
+        .create_nexus(mbus::CreateNexus {
```

Comment:
All of the tests which use the rest client should use the rest types (v0) and not the mbus types as they are not necessarily the same - it's an implementation detail.

Reply:
Done.